docs/home/DEVELOPMENT.md

```diff
@@ -0,0 +1,145 @@
+# Development
+
+<!-- TOC -->
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Preparing the environment](#preparing-the-environment)
+- [Supplying data to the devcontainer](#supplying-data-to-the-devcontainer)
+- [Note for Windows users](#note-for-Windows-users)
+- [Running and debugging](#running-and-debugging)  
+- [Running unit tests](#running-unit-tests)
+
+<!-- /TOC -->
+
+## Introduction
+
+This document will explain how to setup the prefered [VSCode](https://code.visualstudio.com/) development environment. While there are other options to develop Epiphany like [PyCharm](https://www.jetbrains.com/pycharm/) VSCode has the following advantages:
+
+1. Epiphany is developed using many different technologies (Python, Ansible, Terraform, Docker, Jinja, YAML...) and VSCode has good tooling and extensions available to support everything in one IDE.
+
+2. VSCode`s [devcontainers](https://code.visualstudio.com/docs/remote/containers) allow us to quickly setup a dockerized development environment which is the same  for every developer regardless of development platform (Linux, MacOS, Windows).
+
+## Prerequisites
+
+- [VSCode](https://code.visualstudio.com/)
+- Docker-CE
+  - [Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
+  - [MacOS](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+  - [Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+
+Note: VSCode devcontainers are not properly supported using Docker Toolbox on Windows. More info [here](https://github.com/microsoft/vscode-remote-release/issues/95).
```

Comment:
You can consider adding the following info somewhere:

On Windows 7, using Docker Toolbox, in case of an error like `Error response from daemon: invalid mount config for type "bind": invalid mount path: 'd:/github/epiphany' mount path must be absolute.`, you can add the `workspaceMount` property to `core\src\epicli\.devcontainer\devcontainer.json`, for example:
```
"workspaceMount": "src=/d/github/epiphany,dst=/workspaces/epiphany,type=bind"
```
assuming that you already have mounted the path with your local Git repo in the docker-machine (using Virtualbox 'Shared Folders'), for example:
```
docker@default:~$ mount | grep vbox | grep /d
d on /d type vboxsf (rw,nodev,relatime,iocharset=utf8,uid=1000,gid=50)
```

Reply:
I would not add anything regarding Docker Toolbox since its not officially supported and support for Windows 7 is ending in a few months.